src/lightstep-datasource/query_ctrl.js

```diff
@@ -28,6 +30,7 @@ export class LightStepDatasourceQueryCtrl extends QueryCtrl {
     }
 
     this.scope = $scope;
+    this.target.target = this.target.target || DEFAULT_TARGET_VALUE;
```

Comment:
I assume the bug was while it waited for the async API call, it failed on populating anything? So the fix here is setting a default value? LGTM.